server/data/mappers/ciagPlanMapper.ts

```diff
@@ -7,15 +7,19 @@ import SkillsValue from '../../enums/skillsValue'
 import PersonalInterestsValue from '../../enums/personalInterestsValue'
 import InPrisonWorkValue from '../../enums/inPrisonWorkValue'
 import InPrisonEducationValue from '../../enums/inPrisonEducationValue'
+import enumComparator from './enumComparator'
+import previousWorkExperienceDtoComparator from './previousWorkExperienceDtoComparator'
+import futureWorkInterestDtoComparator from './futureWorkInterestDtoComparator'
+import achievedQualificationDtoComparator from './achievedQualificationDtoComparator'
 
 const toCiagPlan = (inductionDto: InductionDto): CiagPlan => {
   return {
     offenderId: inductionDto.prisonNumber,
     desireToWork: inductionDto.workOnRelease.hopingToWork === HopingToGetWorkValue.YES,
     hopingToGetWork: inductionDto.workOnRelease.hopingToWork,
-    reasonToNotGetWork: inductionDto.workOnRelease.notHopingToWorkReasons,
+    reasonToNotGetWork: inductionDto.workOnRelease.notHopingToWorkReasons?.sort(enumComparator),
```

Comment:
Given you're doing this sorting on the outbound `GET` call, I guess we don't need to apply similar sorting on the create/update calls as well. What do you think?

Reply:
100% agreed.
This sorting is 100% a view concern - the order in which we send the data (array values) back to the API in POST or PUT requests is a non-issue 👍

Reply:
Good, I was hoping you were going to say that! 😄